odoo/sql_db.py

```diff
@@ -64,13 +64,20 @@ def flush_env(cr, *, clear=True):
     """ Retrieve and flush an environment corresponding to the given cursor.
         Also clear the environment if ``clear`` is true.
     """
+    env1 = None
```

Comment:
I suggest to use a more explicit variable name for the environment that need to be flushed

```suggestion
    env_to_flush = None
```